sdk/eventhub/event-hubs/src/linkEntity.ts

```diff
@@ -205,7 +201,7 @@ export class LinkEntity {
       this.address
     );
     if (setTokenRenewal) {
-      await this._ensureTokenRenewal();
+      this._ensureTokenRenewal();
```

Comment:
there is one more instance of this, where `await` can be removed, in the managementClient

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/25040/commits/6a3f6dabbf765a9c7af73fe5985d063c1223a403.